exporter/exporterhelper/constants.go

```diff
@@ -28,8 +28,8 @@ var (
 )
 
 const (
-	numDroppedMetricsAttribute  = "num_dropped_metrics"
-	numReceivedMetricsAttribute = "num_received_metrics"
-	numDroppedSpansAttribute    = "num_dropped_spans"
-	numReceivedSpansAttribute   = "num_received_spans"
+	numDroppedTimeSeriesAttribute  = "num_dropped_timeseries"
```

Comment:
This is because we count the number of timeseries not metrics. the reason is that a lot of the systems allow pushing individual timeseries so an receiver/exporter may drop only one timeseries and not the entire metric.

Reply:
OK, makes sense.